specification.md

```diff
@@ -5331,40 +5781,90 @@ Let's take a look at an example. To start, let's define a workflow top-level `re
 
 ```json
 {
-"retries": [
-  {
-    "name": "first-retry-strategy",
-    "delay": "PT1M",
-    "maxAttempts": 5
+  "errors": {
+    "definitions": [
+      {
+        "name": "service-not-available-error",
+        "type": "https://serverlessworkflow.io/spec/errors/communication",
+        "status": 503,
+        "title": "Service Not Available",
+        "detail": "Failed to contact service, even after multiple retries"
+      }
+    ],
+    "handlers": [
+      {
+        "name": "handle-503",
+        "when": [
+          {
+            "status": 503
+          }
+        ],
+        "retry": "retry-five-times",
+        "then": {
+          "throw": {
+            "refName": "service-not-available-error"
+          }
```

Comment:
Even if it's a valid example, I think it might be confusing as the action is already "catching" 503 error to then throw an other one. Maybe `then` could illustrate something else such as a `compensate` or `transition` to `send-error-notification-to-admin` ?
Here, I'm under the impression it might be understood as "you need to create an error definition to handle the said error". 
WDYT @cdavernas @ricardozanini @fjtirado ?

Reply:
Would rethrowing instead address your concern? (not demonstrated anywhere else)

```yaml
then:
  throw: true
```